Tests/UnitTests/Core/Propagator/StraightLineStepperTests.cpp

```diff
@@ -121,11 +123,11 @@ BOOST_AUTO_TEST_CASE(straight_line_stepper_test) {
   StraightLineStepper sls;
 
   // Test the getters
-  BOOST_CHECK_EQUAL(sls.position(slsState), slsState.pos);
-  BOOST_CHECK_EQUAL(sls.direction(slsState), slsState.dir);
-  BOOST_CHECK_EQUAL(sls.momentum(slsState), slsState.p);
-  BOOST_CHECK_EQUAL(sls.charge(slsState), slsState.q);
-  BOOST_CHECK_EQUAL(sls.time(slsState), slsState.t);
+  CHECK_CLOSE_ABS(sls.position(slsState), pos, 1e-6);
+  CHECK_CLOSE_ABS(sls.direction(slsState), dir, 1e-6);
+  CHECK_CLOSE_ABS(sls.momentum(slsState), absMom, 1e-6);
```

Comment:
```suggestion
  CHECK_CLOSE_ABS(sls.position(slsState), pos, eps);
  CHECK_CLOSE_ABS(sls.direction(slsState), dir, eps);
  CHECK_CLOSE_ABS(sls.momentum(slsState), absMom, eps);
```

Reply:
The `momentum(...)` call does actually not always work within `std::numeric_limits<double>::epsilon`.

Reply:
I will leave those as is, because all Steppers will get a new, unified UnitTests once they all look the same.